R/bb_box_score.R

```diff
@@ -0,0 +1,60 @@
+#' Extract Box Scores 
+#'
+#' @param game_id Game ID from a link like: https://stats.ncaa.org/contests/1970309/box_score
+#' @keywords schedule
+#' @export
+#' @examples get_team_schedule(1970309)
+#' @examples get_team_schedule(c(1970309,1806264))
+#'
+#'
+#'
+
+
+bb_box_score <- function(game_id) {
+  all_boxes <- lapply(game_id, function(game_id) {
+    url <- paste0("https://stats.ncaa.org/contests/", game_id,
+                  "/individual_stats")
+    ua <- user_agent("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36")
```

Comment:
since you are calling the same user-agent in both of these. Might be worth setting it as a helper function. 

Also let me test this later to see if it works on Mac, or if there's a broader solution we can try